components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Person from '@material-ui/icons/Person';
+import Icon from '../Icon/Icon';
+import { colors } from '../shared/theme';
+
+const borderStyle = 'border-radius: 50%;';
+
+const AvatarWrapper = styled.a`
+  display: inline-flex;
+  position: relative;
+  height: 24px;
+  text-decoration: unset;
+  color: unset;
+`;
+
+const AvatarImage = styled.img.attrs({
+  alt: 'user picture',
+  ariaHidden: 'true',
+})`
+  ${borderStyle}
+  width: 24px;
+  height: 24px;
+  overflow: hidden;
+`;
+
+const AvatarIcon = styled(Person).attrs({
+  style: {
+    color: colors.neutral[300],
+    fontSize: '18px',
```

Comment:
What do you think to use _em_?

Reply:
this is for the size of the icon, I wanted it to be fixed